app/src/main/java/de/danoeh/antennapod/fragment/PlaybackHistoryFragment.java

```diff
@@ -55,7 +57,20 @@ public boolean onMenuItemClick(MenuItem item) {
             return true;
         }
         if (item.getItemId() == R.id.clear_history_item) {
-            DBWriter.clearPlaybackHistory();
+
+            ConfirmationDialog conDialog = new ConfirmationDialog(
+                    getActivity(),
+                    de.danoeh.antennapod.ui.statistics.R.string.clear_playback_history,
+                    de.danoeh.antennapod.ui.statistics.R.string.clear_playback_history_msg) {
```

Comment:
I think it should be enough to use the already imported R class:

```suggestion
            ConfirmationDialog conDialog = new ConfirmationDialog(getActivity(),
                    R.string.clear_playback_history, R.string.clear_playback_history_msg) {
```